components/Timer.tsx

```diff
@@ -24,7 +24,7 @@ const Display = styled.div<{ isUrgent: boolean }>`
   z-index: 200;
 
   ${(props) =>
-    props.isUrgent &&
+    props.$isUrgent &&
```

Comment:
What's with the `$`?

Reply:
These are transient props, which are props that are meant to only be consumed by `@emotion/styled` and not the underlying DOM node. [You can read about it here](https://styled-components.com/docs/api).

`@emotion/styled` doesn't support it directly so there's a `transientOptions` that I pass in so we can use them.

Reply:
Oh, I see. Thanks!